cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml

```diff
@@ -530,6 +539,7 @@ Resources:
               - 'autoscaling:DescribeAutoScalingInstances'
               - 'autoscaling:CompleteLifecycleAction'
               - 'autoscaling:RecordLifecycleActionHeartbeat'
+              - 'autoscaling:SetInstanceHealth'
```

Comment:
please add
                "ec2:DescribeInstances",
                "ec2:DescribeTags",

Reply:
are you asking for both asg policy and asg policy for lambda to add these two actions ?